app/upgrades/v2_5_0/v2_5_0.go

```diff
@@ -53,145 +51,44 @@ var Upgrade = upgrades.Upgrade{
 	StoreUpgrades: storetypes.StoreUpgrades{},
 }
 
-// Set of addresses that held stNIBI in ERC20 form prior to the v2.5.0 upgrade
-func MAINNET_STNIBI_HOLDERS() []gethcommon.Address {
-	holderStrs := set.New(
-		"0x603871c2ddd41c26Ee77495E2E31e6De7f9957e0",
-		"0x447fC0471f5d8150A790e19000930108aCCe0BC7",
-		"0x2DD37531749e1AF248720fe8F9D1A51517D9748F",
-		"0x07A1D54115D5B535F298231Cc6EaA360Bc4667f9",
-		"0x5361EBC7CF6689565EC4C380b7793e238ad6Be45",
-	)
-	holders := make([]gethcommon.Address, len(holderStrs))
-	for idx, h := range holderStrs.ToSlice() {
-		holders[idx] = gethcommon.HexToAddress(h)
-	}
-	return holders
-}
-
-var (
-	// MAINNET_STNIBI_ADDR is the (real) hex address of stNIBI on mainnet.
-	MAINNET_STNIBI_ADDR = gethcommon.HexToAddress("0xcA0a9Fb5FBF692fa12fD13c0A900EC56Bb3f0a7b")
-
-	// MAINNET_NIBIRU_SAFE_ADDR: Address of a Gnosis Safe managed by the Nibiru team
-	// See https://nibiscan.io/address/0x22CBd7CbF3b33681abB3Ced4D64d71acB9a9dCd2/contract/6900/code
-	MAINNET_NIBIRU_SAFE_ADDR = gethcommon.HexToAddress("0x22CBd7CbF3b33681abB3Ced4D64d71acB9a9dCd2")
```

Comment:
Likewise, we no longer need to keep track of excess balances and send them to the Nibiru Gnosis Safe because everybody's balances will be maintained after the upgrade.